TestControllers/TestController.psm1

```diff
@@ -772,75 +804,77 @@ Class TestController
 
-					if ($gcovCollected) {
-						Write-LogInfo "GCOV data collected successfully"
-					} else {
-						Write-LogErr "Failed to collect GCOV data from VM: $($VM.RoleName)"
+						if ($gcovCollected) {
+							Write-LogInfo "GCOV data collected successfully"
+						} else {
+							Write-LogErr "Failed to collect GCOV data from VM: $($VM.RoleName)"
+						}
 					}
-				}
 
-				Run-LinuxCmd -ip $VM.PublicIP -port $VM.SSHPort -runAsSudo `
-					-username $global:user -password $global:password `
-					-command "dmesg > ./${currentBootLogFile}" | Out-Null
-				Copy-RemoteFiles -download -downloadFrom $VM.PublicIP -port $VM.SSHPort -files "./${currentBootLogFile}" `
-					-downloadTo $BootLogDir -username $global:user -password $global:password | Out-Null
-				Write-LogInfo "$($VM.RoleName): $status kernel log, ${currentBootLogFile}, collected successfully."
-
-				Write-LogInfo "Checking for call traces in kernel logs.."
-				$KernelLogs = Get-Content $currentBootLog
-				$callTraceFound  = $false
-				foreach ($line in $KernelLogs) {
-					if (( $line -imatch "Call Trace" ) -and ($line -inotmatch "initcall ")) {
-						Write-LogErr $line
-						$callTraceFound = $true
-					}
-					if ($callTraceFound) {
-						if ($line -imatch "\[<") {
+					Run-LinuxCmd -ip $VM.PublicIP -port $VM.SSHPort -runAsSudo `
+						-username $global:user -password $global:password `
+						-command "dmesg > ./${currentBootLogFile}" | Out-Null
+					Copy-RemoteFiles -download -downloadFrom $VM.PublicIP -port $VM.SSHPort -files "./${currentBootLogFile}" `
+						-downloadTo $BootLogDir -username $global:user -password $global:password | Out-Null
+					Write-LogInfo "$($VM.RoleName): $status kernel log, ${currentBootLogFile}, collected successfully."
+
+					Write-LogInfo "Checking for call traces in kernel logs.."
+					$KernelLogs = Get-Content $currentBootLog
+					$callTraceFound  = $false
+					foreach ($line in $KernelLogs) {
+						if (( $line -imatch "Call Trace" ) -and ($line -inotmatch "initcall ")) {
 							Write-LogErr $line
+							$callTraceFound = $true
+						}
+						if ($callTraceFound) {
+							if ($line -imatch "\[<") {
+								Write-LogErr $line
+							}
 						}
 					}
-				}
-				if (!$callTraceFound) {
-					Write-LogInfo "No kernel call traces found in the kernel log"
-				}
+					if (!$callTraceFound) {
+						Write-LogInfo "No kernel call traces found in the kernel log"
+					}
 
-				if($status -imatch "Final") {
-					$ret = Compare-OsLogs -InitialLogFilePath $InitialBootLog -FinalLogFilePath $currentBootLog -LogStatusFilePath $KernelLogStatus `
-						-ErrorMatchPatten "fail|error|warning"
+					if($status -imatch "Final") {
+						$ret = Compare-OsLogs -InitialLogFilePath $InitialBootLog -FinalLogFilePath $currentBootLog -LogStatusFilePath $KernelLogStatus `
+							-ErrorMatchPatten "fail|error|warning"
 
-					if ($ret -eq $false) {
-						$retValue = $false
-					}
+						if ($ret -eq $false) {
+							$retValue = $false
+						}
```

Comment:
indent?

Reply:
Test Controller using 'tab' as indent, and already consistent with previous indenting, no need to update here.